.storybook/components/TokenSpecimen/TokenSpecimen.js

```diff
@@ -86,11 +86,10 @@ export class TokenSpecimen extends Component {
 }
 
 TokenSpecimen.propTypes = {
-  listItems: PropTypes.arrayOf(
-    PropTypes.shape({
-      name: PropTypes.string.isRequired,
-      value: PropTypes.any.isRequired,
-      comment: PropTypes.string,
-    }),
-  ).isRequired,
+  name: PropTypes.string.isRequired,
+  value: PropTypes.any.isRequired,
+  comment: PropTypes.string,
+  inlineStyles: PropTypes.any,
```

Comment:
Does `PropTypes.object` work here? I'm reading some conflicting things about it working and not